webdriver/tests/bidi/browsing_context/print/invalid.py

```diff
@@ -133,6 +133,20 @@ async def test_params_page_invalid_value(bidi_session, top_context, page):
         )
 
 
+@pytest.mark.parametrize(
+    "page",
+    [
+        {"height": 0.01},
+        {"width": 0.01},
+    ],
+)
+async def test_params_page_small_value(bidi_session, top_context, page):
+    with pytest.raises(error.UnsupportedOperationException):
```

Comment:
Given the we have defaults in the CDDL and define a lower bound the CDDL parser should actually fail on this value and WebDriver needs to return with a `invalid argument` error. 

That said you should better combine it with the previous test and extend the parameterized value list.

Reply:
Agreed! Done